libraries/apollo-compiler/src/test/graphql/com/example/case_sensitive_enum/java/operationBased/case_sensitive_enum/type/Enum.java.expected

```diff
@@ -60,15 +61,15 @@ public class Enum {
         return true;
       }
       if (o instanceof UNKNOWN__) {
-        return true;
+        return super.equals(o);
```

Comment:
What is the rationale for this?

Reply:
I noticed that without this, 2 unknown enums with a different rawValue would still be equal, since that field is not included in the comparison. The super version does include the field in the comparison.

Reply:
Shouldn't it compare `rawValue`s instead? I would expect `Object.equals()` to do referential equality only?

Reply:
You're totally right! I assumed equals was implemented in the base class but it's not, so calling super doesn't work. In the Kotlin version the equals/hashCode is ad-hoc, I'll do the same in Java.